### DISCUSSION TITLE
New user not able to connect

### DISCUSSION THREAD

**Discussion by BrianGarland** [codefori/vscode-ibmi#2351] [General]
I have a new developer that cannot connect to our IBM i server.  They can sign on via green screen just fine.  

We are using user/password for connection.  Several uses can connect but some new hires cannot.   

"Not connected to xxxxx! Check your credentials (All configured authentication methods failed)." 

There are no details under this message.  Is there a place where I can determine why?

**Comment by worksofliam** [comment]
@BrianGarland Have you tried getting them to connect via regular SSH?

**Comment by BrianGarland** [comment]
Getting "Permission denied, please try again.  "

**Comment by BrianGarland** [comment]
Looks like our IT admin has things pretty locked down.  We're checking with them.

**Comment by BrianGarland** [comment]
For completion of my Code for IBM i connection question:   We had to add the new users to sshd_config AllowUsers directive.   There must be a better way to do this, but that's for another forum.

**Reply by chrjorgensen** [reply]
@BrianGarland SSH is a well-known attack vector, so you should certainly have restricted the number of users allowed through this interface.

We use a group profile called `GRPSSH` and the `AllowGroups` directive to allow SSH access, and any user requiring SSH access (for system administration, use of open source tools or development in Code for IBM i) must be a member of this group.

We followed [these guidelines](https://www.mcpressonline.com/security/ibm-i-os400-i5os/controlling-access-to-ssh-on-ibm-i) from Carol Woodbury.
